hr_timesheet_sheet_current/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "version": "12.0.1.0.0",
     "license": "AGPL-3",
     "author": "Coop IT Easy SC",
-    "website": "https://coopiteasy.be",
+    "website": "https://github.com/coopiteasy/cie-timesheet",
```

Comment:
i don’t think that this should be changed. the github repo’s url is used only for oca repos. this should be fixed in `.copier-answers.yml`.

Reply:
fixed (in module and in copier)